config/jail.conf

```diff
@@ -206,6 +206,23 @@ action_badips = badips.py[category="%(__name__)s", banaction="%(banaction)s", ag
 # Report ban via badips.com (uses action.d/badips.conf for reporting only)
 #
 action_badips_report = badips[category="%(__name__)s", agent="%(fail2ban_agent)s"]
+# Actions to report to abuseipdb.com via API.
+# See action.d/abuseipdb.conf and https://wiki.shaunc.com/wikka.php?wakka=ReportingToAbuseIPDBWithFail2Ban
+# If you want cleaner reports that ensure no user data see the helper script at the above IP
+# NOTE: These reports may include sensitive Info.
+
+# IMPORTANT: Register for abuseipdb [https://www.abuseipdb.com], get, and set the api key at the bottom of action.d/abuseipdb.conf
```

Comment:
This lot of parameters burdened the jail substitution, therefore belong imho into corresponding action...